config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get 'posts/index'
+  get 'posts/show'
+  get 'users/index'
+  get 'users/show'
```

Comment:
- [ ] Your application routes should be similar to that of the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md) provided. The routes in the wireframe look like these:
- http://localhost:3000/users/
- http://localhost:3000/users/745
- http://localhost:3000/users/745/posts
- http://localhost:3000/users/posts/3

#### Hint: To be able to create routes in this fashion, I will advise you to use the Rails resources as specified in this [article](https://guides.rubyonrails.org/routing.html#:~:text=2.7%20Nested%20Resources) to create your routes. Please, only create the needed routes. 👍 😎